src/main/java/ru/yandex/qatools/allure/repository/XmlConverter.java

```
@@ -0,0 +1,45 @@
+package ru.yandex.qatools.allure.repository;
+
+import retrofit.converter.ConversionException;
+import retrofit.converter.Converter;
+import retrofit.mime.TypedInput;
+import retrofit.mime.TypedOutput;
+
+import javax.xml.bind.*;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.lang.reflect.Type;
+
+/**
+ * eroshenkoam
+ * 31.07.15
+ */
+public class XmlConverter implements Converter {
+
+    @Override
+    public Object fromBody(TypedInput typedInput, Type type) throws ConversionException {
+        Object result;
+
+        try {
+            result = JAXB.unmarshal(typedInput.in(), (Class) type);
+        } catch (IOException e) {
+            throw new ConversionException(e);
+        }
+
+        try {
+            typedInput.in().close();
+        } catch (IOException e) {
+            throw new ConversionException(e);
+        }
+
+        return result;
+    }
+
+    @Override
+    public TypedOutput toBody(Object o) {
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
```

You can just throw `UnsupportedOperationException`